assets/css/overrides.css

```diff
@@ -44,3 +44,57 @@ content-container class */
 .news-container, .hero-container {
     width: 100%;
 }
+
+/* Admonitions
+
+Rely on the theme's admonition styling. Only apply minimal readability fixes
+so content remains accessible in dark mode across themes.
+*/
+.adm { opacity: 1; }
+.adm a { text-decoration: underline; }
+
+@media (prefers-color-scheme: dark) {
+  /* Ensure admonitions stay readable even if the theme uses light backgrounds */
+  .article .adm, .article .adm *,
+  .content .adm, .content .adm *,
+  .markdown .adm, .markdown .adm * {
+    color: var(--text, #f1f5f9) !important;
+  }
+  .adm a { color: var(--primary, #9ae6b4) !important; }
+  /* Provide a gentle dark surface without hardcoding brand colors */
+  .adm { background: var(--surface-2, rgba(255,255,255,0.06)) !important; }
+}
```

Comment:
Instead of `!important`s, we should address any dark theme problem upstream with the theme itself – I would advise against using them unless they're really needed